edx_django_utils/cache/README.rst

```diff
@@ -57,11 +57,31 @@ Sample Usage (cache miss)::
         # calculate x, set in cache, and return value.
     return x_cached_response.value
 
-**Warning**: When storing a bool, `Memcached will return an int`_.
-However, the RequestCache will return a bool. The first time a value is set
-the TieredCache may return a bool and in later requests the TieredCache may
-return an int. Therefore, do not check if the value ``is False`` or
-``is True`` explicitly.
+Warning when storing bools
```

Comment:
@cpennington: Does this update address your concerns?

Reply:
Yeah, looks good.